packages/synapse-bridge/src/patterns/HeaderBar/types.d.ts

```diff
@@ -0,0 +1,7 @@
+import { RawLocation } from "vue-router";
+
+export interface NavigationItem {
+	href?: string;
+	label: string;
```

Comment:
```suggestion
	label: string;
	href?: string;
```
Histoire d'avoir les propriétés obligatoires en premier